src/auto4_base.cpp

```diff
@@ -322,10 +322,14 @@ namespace Automation4 {
 
 		std::vector<std::future<std::unique_ptr<Script>>> script_futures;
 
+		std::set<std::filesystem::path> dirnames;
 		for (auto tok : agi::Split(path, '|')) {
 			auto dirname = config::path->Decode(std::string(tok));
 			if (!agi::fs::DirectoryExists(dirname)) continue;
 
+			if (dirnames.count(dirname)) continue;
```

Comment:
I don't love this but I suppose it's generally possible for paths to be included twice due to someone changing settings, so it's probably for the best.

The duplication causing crashes is of separate concern.

Reply:
The reason for this was that, on portable builds, `?user` ends up being equal to `?data`, so the default autoload path (`?data/automation/autoload|?user/automation/autoload`) ends up being the same directory twice.

The actual crashes can show a number of different possible error messages, so there might be multiple issues. One day I'd like to sit down and debug this in more detail but for now this stops portable builds from sometimes crashing on startup.